src/groovy/util/ConfigHelper.groovy

```diff
@@ -29,4 +30,16 @@ class ConfigHelper {
 
     }
 
+    static findAccessRule(String controllerName, String actionName, Object rules) {
```

Comment:
I think we could (should) pull the rules in via the ConfigurationHolder.
```
List rules = ConfigurationHolder.config.openboxes.security.rbac.rules?:[]
```
Can't remember if the rules are a list or a map so I'll leave that for you to figure out.

Reply:
Oh I remember why you said you needed to do this ... tests. Even unit tests can mock configuration properties.
```
class UserTests extends GrailsUnitTestCase {
    @Before
    void setUp() {
        super.setUp()
        mockConfig("openboxes.anonymize.enabled = false")
    }
```

Reply:
thanks